docs/webhooks/index.html

```diff
@@ -967,6 +972,51 @@ <h3 id="certificationRequestsMerged">certificationRequestsMerged</h3>
     deletedRequestId: 'lehjeh36dleh29',
     mergedRequestId: 'dhey29hcl28hl28',
   }
+};</code></pre>
+            </div>
+            <h2>Events for CodeSignal Live Interview</h2>
+            <div class="subsection">
+              <h3 id="liveinterviewworkflow">Live Interview webhooks process flow</h3>
+              <img
+                alt="Diagram of webhooks process flow for CodeSignal Live Interview"
+                width="100%"
+                src="../images/live-interview-events.svg"
+              />
+            </div>
+            <div class="subsection">
+              <h3 id="liveInterviewFinished">liveInterviewFinished</h3>
+              <p>
+                Fired when a interviewer finishes a live interview or when a live interview timeout
+              </p>
+              <div>Request format:</div>
+              <pre><code>{
+  event: 'liveInterviewFinished',
+  triggeredOn: number,
+  payload: {
+    interviewId: LiveInterviewId;
+    candidateName: Maybe<string>;
+    candidateEmail: Maybe<string>;
+    startDate: Maybe<number>;
+    endDate: Maybe<number>;
+    questionsSolved: number;
+    questionsAttempted: number;
+    url: string;
+  }
```

Comment:
`Maybe<string>` type is not a standard type, so it might be confusing to consumers. Additionally, this doesn't render correctly:
![image](https://github.com/user-attachments/assets/9a2d862e-487e-406e-9638-59966e0e257f)

There's also in inconsistency between `,` and `;` in the definition.

```suggestion
              <pre><code>{
  event: 'liveInterviewFinished',
  triggeredOn: number,
  payload: {
    interviewId: string,
    candidateName: string,
    candidateEmail: string,
    startDate: number,
    endDate: number,
    questionsSolved: number,
    questionsAttempted: number,
    url: number
  }
```

I've also changed a few types:
* For name & email, let's always return a string. It can be empty if we don't have this information.
* Both `startDate` and `endDate` will be defined, since this event will be fired only for interviews that have been started and are now over.